### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by yohannesdagnachew**
### Description
- Create an npm project with webpack  ✅ 
- Use Gitflow ✅ 
- Write plain HTML markup with minimum styling ✅

### CODE REVIEW

**Comment by medaminedev66** [comment]
Hi @yohannesdagnachew, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a `node_modules/` folder. Please ignore the `node_modules/`
 folder using the .gitignore file. The `node_modules/` folder is the folder where all packages required by your JavaScript project are downloaded and installed. This folder is commonly excluded from a remote repository because it has a large size and you shouldn’t add code that you didn’t write to the repository.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
...
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Have a good working knowledge of Git and Github
+### Setup
+- Open a command line terminal (Git bash preferrably) and navigate to a directory where you would like to save the work folder using cd.
+- Clone the repository with git clone https://github.com/wale-prog/To-do-list.git
+-To get the cuurent working tree, git fetch --all, then checkout to "todo-list-structure" branch.
```
Comment (elmar8287) [inline README.md:29]: - [ ] [OPTIONAL] Please, consider fixing the correct repo link - it seams this link is came from the old project "To do list"

File: README.md [hunk 2]
```diff
@@ -1 +1,69 @@
...
+
+- GitHub: [@githubhandle](https://github.com/yohannesdagnachew)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/yohannes-dagnachew-5b163a236/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [https://github.com/yohannesdagnachew/To-Do-List/issues).
```
Comment (elmar8287) [inline README.md:55]: - [ ] [OPTIONAL] Please, consider to fix the correct repo link - it seams this link is came from the old project "To do list"

File: README.md [hunk 3]
```diff
@@ -1 +1,69 @@
...
+
+## Acknowledgments
+
+- MY Friends
+- Microverse former student
+- etc
+
+## 📝 License
+
+This project is [Microverse](./MIT.md) licensed.
```
Comment (elmar8287) [inline README.md:69]: - [ ] [OPTIONAL] Please, consider fixing the broken link to the MIT.md file.

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### OPTIONAL suggestions in comments under the review




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**